tools/ci_build/github/azure-pipelines/templates/react-native-ci.yml

```diff
@@ -380,7 +380,7 @@ stages:
 
     - task: PublishPipelineArtifact@1
       inputs:
-        artifact: e2e_test_logs_$(Build.BuildId)_$(Date:yyyyMMddHHmmss)
+        artifact: e2e_test_logs_$(Build.BuildId)_$(Build.BuildNumber)
```

Comment:
will this be updated for job retries within a single build?
`$(System.JobAttempt)` looks useful here

Reply:
ah good suggestion - build number probably won't be different between retries. added a suffix for jobattempt.